code/stable/gamephysics/SRS/GamePhysics_SRS.tex

```diff
@@ -511,14 +514,14 @@ \subsubsection{General Definitions}
         
 \\ \midrule \\
 Equation & \begin{displaymath}
-           \mathbf{g}=-\frac{G M}{\mathbf{r}^{2}} \mathbf{\hat{r}}
+           \mathbf{g}=-\frac{G M}{\text{||}\mathbf{r}\text{||}^{2}} \mathbf{\hat{r}}
            \end{displaymath}
 \\ \midrule \\
 Description & \begin{symbDescription}
               \item{$\mathbf{g}$ is the gravitational acceleration ($\frac{\text{m}}{\text{s}^{2}}$)}
               \item{$G$ is the gravitational constant ($\frac{\text{m}^{3}}{(\text{kg}\text{s}^{2})}$)}
               \item{$M$ is the mass of the larger rigid body (kg)}
-              \item{$\mathbf{r}$ is the displacement (m)}
+              \item{$\text{||}\mathbf{r}\text{||}$ is the Euclidean norm of the displacement (m)}
```

Comment:
Why is r called displacement?  I believe we have discussed this before, but I do not remember the explanation.